docs/ossm/README.md

```diff
@@ -0,0 +1,196 @@
+[Return to Project Root](../../)
+
+# Table of Contents
+
+- [User Documentation](#user-documentation)
+- [istio-csr Integration](#istio-csr-integration)
+  - [Common Setup](#common-setup)
+  - [InPlace Strategy](#inplace-strategy)
+  - [RevisionBased Strategy](#revisionbased-strategy)
+  - [Verify](#verify)
+  - [RevisionBased Upgrades](#upgrades---revisionbased)
+
+# User Documentation
+
+This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the [Sail Operator documentation](../README.md). If you are using the OpenShift Service Mesh 3 operator, you should consider this documentation the source of truth. If there is not documentation here for a particular topic, follow the [Sail Operator documentation](../README.md).
```

Comment:
I'm a bit confused if we should call this documentation as the source of truth for OSSM 3.0 over the product docs.

Whats your opinion on the following rephrased version?

This documentation is specific to the OpenShift Service Mesh product and may differ slightly from the Sail Operator documentation. For OpenShift Service Mesh 3, this documentation should be prioritized over the Sail Operator documentation. However, the official source of truth is the product documentation available at [docs.openshift.com](https://docs.openshift.com/service-mesh/3.0.0tp1/about/ossm-about-openshift-service-mesh.html). If a particular topic is not covered here, you can refer to the Sail Operator documentation as a supplementary resource.

Reply:
Agreed... I would only add one small addition to also reference the istio.io docs. For the last sentence, change to "If a particular topic is not covered here, you can refer to the Sail Operator or Istio.io documentation as a supplementary resource."

Reply:
That's better. By saying "source of truth", I wanted to emphasis `Sail Operator docs < these docs` but you're right it should be `Sail Operator docs < these docs < product docs`.